[CI] MixedClusterClientYamlTestSuiteIT test {p0=search.vectors/260_knn_lookup_query_builder/Test error when document not found} failing

**Build Scans:**
- [elasticsearch-intake #38097 / 9.3.4_bwc-snapshots](https://gradle-enterprise.elastic.co/s/3266ggxyptizg)
- [elasticsearch-pull-request #139165 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/e4ht4gwc762mw)
- [elasticsearch-pull-request #139164 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/hwadoobhg2me2)
- [elasticsearch-pull-request #139161 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/m62fynb6r6ufg)
- [elasticsearch-intake #38094 / 9.3.4_bwc-snapshots](https://gradle-enterprise.elastic.co/s/ojuer5bw4r4nu)
- [elasticsearch-intake #38093 / 9.3.4_bwc-snapshots](https://gradle-enterprise.elastic.co/s/x4exgcqrfntqg)
- [elasticsearch-pull-request #139158 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/dtswpgomdut26)
- [elasticsearch-pull-request #139157 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/tyu7fue6bszma)
- [elasticsearch-pull-request #139154 / 9.3.4_part_6_bwc-snapshots](https://gradle-enterprise.elastic.co/s/7pyhkx4t6qay6)
- [elasticsearch-intake #38089 / 9.3.4_bwc-snapshots](https://gradle-enterprise.elastic.co/s/6hhd3kfah4fpi)

**Reproduction Line:**
```
./gradlew ":qa:mixed-cluster:v9.3.4#mixedClusterTest" -Dtests.class="org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT" -Dtests.method="test {p0=search.vectors/260_knn_lookup_query_builder/Test error when document not found}" -Dtests.seed=230DE374FFB9212 -Dtests.bwc=true -Dtests.locale=tr -Dtests.timezone=Europe/Podgorica -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('test%20%7Bp0%3Dsearch.vectors%2F260_knn_lookup_query_builder%2FTest%20error%20when%20document%20not%20found%7D'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: java.lang.AssertionError: expected [404] status code but api [search] returned [400 Bad Request] [{"error":{"root_cause":[{"type":"named_object_not_found_exception","reason":"[1:42] unknown field [lookup]","stack_trace":"org.elasticsearch.ElasticsearchException$1: [1:42] unknown field [lookup]\n\tat org.elasticsearch.server@9.3.4-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:799)\n\tat org.elasticsearch.server@9.3.4-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:795)\n\tat org.elasticsearch.server@9.3.4-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:795)\n\tat org.elasticsearch.server@9.3.4-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:713)\n\tat org.elasticsearch.server@9.3.4-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:202)\n\tat org.elasti
[truncated]
```

**Issue Reasons:**
- [main] 3 consecutive failures in test test {p0=search.vectors/260_knn_lookup_query_builder/Test error when document not found}
- [main] 2 consecutive failures in step 9.3.4_bwc-snapshots
- [main] 3 consecutive failures in step 9.3.4_part_6_bwc-snapshots
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 2 consecutive failures in pipeline elasticsearch-pull-request
- [main] 183 failures in test test {p0=search.vectors/260_knn_lookup_query_builder/Test error when document not found} (30.3% fail rate in 603 executions)
- [main] 57 failures in step 9.3.4_bwc-snapshots (52.8% fail rate in 108 executions)
- [main] 126 failures in step 9.3.4_part_6_bwc-snapshots (64.6% fail rate in 195 executions)
- [main] 57 failures in pipeline elasticsearch-intake (52.8% fail rate in 108 executions)
- [main] 126 failures in pipeline elasticsearch-pull-request (64.6% fail rate in 195 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -352,12 +352,6 @@ tests:
 - class: org.elasticsearch.xpack.esql.datasources.ExternalSourceDrainUtilsTests
   method: testBudgetAccountingAcrossAsyncBoundaries
   issue: https://github.com/elastic/elasticsearch/issues/147065
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.vectors/260_knn_lookup_query_builder/Test lookup base64 query vector builder with exclude_source_vectors false}
-  issue: https://github.com/elastic/elasticsearch/issues/147069
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.vectors/260_knn_lookup_query_builder/Test lookup base64 query vector builder with exclude_source_vectors true}
-  issue: https://github.com/elastic/elasticsearch/issues/147070
 - class: org.elasticsearch.xpack.esql.datasources.ExternalSourceDrainUtilsTests
   method: testCancellationWhileWaitingOnWaitForSpace
   issue: https://github.com/elastic/elasticsearch/issues/147073
@@ -373,12 +367,6 @@ tests:
 - class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
   method: test {p0=indices.simulate_index_template/10_basic/Simulate index template with data stream with mapping and setting overrides}
   issue: https://github.com/elastic/elasticsearch/issues/147064
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.vectors/260_knn_lookup_query_builder/Test error when field is missing}
-  issue: https://github.com/elastic/elasticsearch/issues/147076
-- class: org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT
-  method: test {p0=search.vectors/260_knn_lookup_query_builder/Test error when document not found}
-  issue: https://github.com/elastic/elasticsearch/issues/147081
 - class: org.elasticsearch.xpack.rank.rrf.RRFRankClientYamlTestSuiteIT
   method: test {yaml=rrf/310_rrf_retriever_simplified/Expansion to equivalent custom sub-retrievers returns the same results}
   issue: https://github.com/elastic/elasticsearch/issues/146464

--- a/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/260_knn_lookup_query_builder.yml
+++ b/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/260_knn_lookup_query_builder.yml
@@ -1,6 +1,6 @@
 setup:
   - requires:
-      cluster_features: "mapper.base64_dense_vectors"
+      cluster_features: ["mapper.base64_dense_vectors", "search.vectors.lookup_query_vector_builder"]
       reason: 'search.vectors.lookup_query_vector_builder'
 
 # one index with excluded vectors, with bytes, floats, bits and bfloats16

